judo-ui-typescript-rest-api/src/main/resources/rest/queryCustomizer.ts.hbs

```diff
@@ -1,13 +1,13 @@
 {{> fragment.header.hbs }}
 
 import type { QueryCustomizer } from '@judo/data-api-common';
-import { {{ getClassName classType }} } from '../model/{{ getClassName classType }}';
+import { {{ classDataName classType "" }} } from '../model/{{ classDataName classType "" }}';
```

Comment:
Wouldn't it be simpler if there was a parameter list for `classDataName` without the last parameter?